packages-exp/remote-config-exp/src/api.ts

```diff
@@ -44,7 +44,7 @@ export function getRemoteConfig(app: FirebaseApp = getApp()): RemoteConfig {
 
 /**
  * Makes the last fetched config available to the getters.
- * @param remoteConfig - the remote config instance
+ * @param remoteConfig - The remote config instance.
```

Comment:
If we're referring to the product, it should be capped, "Remote Config," here and throughout.

But is it actually a literal, like " `RemoteConfig` instance "  ?